File: frontend/src/assets/two sum/two_sum.py
def twoSum(nums, target):
    for i in range(len(nums)):
        for j in range(i + 1, len(nums)):
            if nums[i] + nums[j] == target:
                return [i, j]
    return []


if __name__ == "__main__":
    t = int(input())
    for _ in range(t):
        n = int(input())
        v = list(map(int, input().split()))
        target = int(input())
        ans = twoSum(v, target)
        print(*ans)

File: frontend/src/assets/two sum/two_sum.cpp

#include <bits/stdc++.h>
using namespace std;
vector<int> twoSum(vector<int> &nums, int target)
{
    for (int i = 0; i < nums.size(); i++)
    {
        for (int j = i + 1; j < nums.size(); j++)
        {
            if (nums[i] + nums[j] == target)
            {
                vector<int> v;
                v.push_back(i);
                v.push_back(j);
                return v;
            }
        }
    }
    vector<int> v;
    return v;
}

int main()
{
    int t;
    cin >> t;
    while (t--)
    {
        int n;
        cin >> n;
        vector<int> v(n);
        for (int i = 0; i < n; i++)
        {
            cin >> v[i];
        }
        int target;
        cin >> target;
        vector<int> ans = twoSum(v, target);
        for (int i = 0; i < ans.size() - 1; i++)
        {
            cout << ans[i] << " ";
        }
        cout << ans[ans.size() - 1];
        cout << '\n';
    }
}
